applications/GeoMechanicsApplication/tests/cpp_tests/custom_strategies/test_geomechanics_time_integration_scheme.cpp

```diff
@@ -53,8 +53,10 @@ class GeoMechanicsSchemeTester
         mModel.CreateModelPart("dummy", 2);
     }
 
-    template <class T>
-    void TestFunctionCalledOnComponent_IsCalledOnActiveAndInactiveComponents()
+    template <class T, typename AddComponentToModelPartLambda>
```

Comment:
Perhaps we can make this type name slightly more general? For instance:
```suggestion
    template <typename T, typename AddComponentToModelPartCallable>
```

Reply:
Done